packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -195,6 +195,11 @@ export interface BrowserTracingOptions {
    * Default: (url: string) => true
    */
   shouldCreateSpanForRequest?(this: void, url: string): boolean;
+
+  /**
+   * Is called when spans are started for outgoing requests.
+   */
```

Comment:
since this is user-facing, I recommend we make JSDoc a bit more explicit. Feel free to go with whatever you think is finde but here's a suggestion:

```suggestion
  /**
   * This callback is invoked directly after a span is started for an outgoing fetch or XHR request.
   * You can use it to annotate the span with additional data or attributes, for example by setting
   * attributes based on the passed request headers.
   */
```